core/src/main/scala/org/http4s/blaze/channel/nio1/NIO1HeadStage.scala

```diff
@@ -214,27 +221,28 @@ private[nio1] abstract class NIO1HeadStage(
       }
 
       writeData = null
+      try ch.close()
+      catch {
+        case ex: IOException => logger.debug(ex)("During close")
```

Comment:
debug?

Reply:
Debug seemed appropriate considering we're trying to end the connection. Honestly I don't know why that method can throw at all: if it throws and won't close, there is no way to release the resource, and if it closes and throws, what was the point of throwing?

Reply:
I'm concerned about masking an an underlying problem.  We've done all we can, and the caller can't do more, so I agree it's not a throw.  It shouldn't affect the user experience, so I agree it's not a `log.error`.  But something is wrong that I would want to understand, so I think it's a `log.warn`.

Reply:
Fair enough, I'll make it a `warn`.